js/src/bitget.js

```diff
@@ -4314,10 +4314,12 @@ export default class bitget extends Exchange {
      * @param {boolean} [params.oneWayMode] *swap and future only* required to set this to true in one_way_mode and you can leave this as undefined in hedge_mode, can adjust the mode using the setPositionMode() method
      * @param {bool} [params.hedged] *swap and future only* true for hedged mode, false for one way mode, default is false
      * @param {bool} [params.reduceOnly] true or false whether the order is reduce-only
+     * @param {string} [params.tradeSide] *future only* Trade type, only required in hedge mode: `"open"` for opening a position, `"close"` for closing a position.
```

Comment:
you don't need this if you provide hedged:True in `params`

Reply:
hedged: true is not enough in bitget. While trying to create sell order with already existing long position, it will open new short instead of closing long. tradeSide resolved this issue for me

Reply:
@OleksiiPaliiUA But in that case, you need to also provide `reduceOnly: true` in `params`